components/Stats.js

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<ul className='font-lexend text-5xl decoration-black font-semibold absolute left-6 bottom-11 leading-15 flex items-center text-center space-x-32'>
+				<li>10 members</li>
```

Comment:
<img width="937" alt="image" src="https://user-images.githubusercontent.com/36285610/184819133-0cacb55a-de5d-45e4-aad8-52ac885d0038.png">

Don't use the `ul` and `li` tags since they aren't very responsive, they force you to scroll sideways if the screen is too small. You can instead use Bootstrap Grid to handle how it responds to different sizes.

Reply:
Also try to have it such that the numbers and text are on the same line. Can we also make it such that it is 10+ members ...